setup.py

```diff
@@ -31,12 +31,17 @@
 if __name__ == '__main__':
     setup(
         name='pyensembl',
-        version="0.5.3",
+        version="0.5.4",
         description="Python interface to ensembl reference genome metadata",
       	author="Alex Rubinsteyn",
         author_email="alex {dot} rubinsteyn {at} mssm {dot} edu",
       	url="https://github.com/hammerlab/pyensembl",
         license="http://www.apache.org/licenses/LICENSE-2.0.html",
+        entry_points={
+          'console_scripts': [
```

Comment:
It sticks an executable `pyensembl` into the Python install dir, and `pyensembl` looks like:

```
#!/Users/tavi/.virtualenvs/pyensembl/bin/python2.7
# EASY-INSTALL-ENTRY-SCRIPT: 'pyensembl==0.5.4','console_scripts','pyensembl'
__requires__ = 'pyensembl==0.5.4'
import sys
from pkg_resources import load_entry_point

if __name__ == '__main__':
    sys.exit(
        load_entry_point('pyensembl==0.5.4', 'console_scripts', 'pyensembl')()
    )
```